sdks/java/io/hadoop/jdk1.8-tests/pom.xml

```diff
@@ -114,6 +114,16 @@ and overrides the enforcer rules to allow 1.8 only behavior without making all o
           </dependency>
         </dependencies>
       </plugin>
+      <plugin>
```

Comment:
this was deliberately removed. Can you explain why it was added back? 

Are you not running with -Pio-it?

(update: I realized I'm not sure whether or not you ever actually removed it. Either way - it should be there and it causes problems.)